connect-rest-extension/src/main/java/io/debezium/kcdrestextension/DebeziumConnectRestExtensions.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.kcdrestextension;
+
+import java.util.Map;
+
+import org.apache.kafka.common.utils.AppInfoParser;
+import org.apache.kafka.connect.rest.ConnectRestExtension;
+import org.apache.kafka.connect.rest.ConnectRestExtensionContext;
+
+public class DebeziumConnectRestExtensions implements ConnectRestExtension {
```

Comment:
Why the plural? I'd suggest `DebeziumConnectRestExtension` instead.

Reply:
typo. will fix it. good catch